looper/cli_pydantic.py

```diff
@@ -167,16 +241,112 @@ def run_looper(args: TopLevelParser, parser: ArgumentParser):
                 )
                 raise
 
+        if subcommand_name in ["runp"]:
+            compute_kwargs = _proc_resources_spec(subcommand_args)
+            collate = Collator(prj)
+            collate(subcommand_args, **compute_kwargs)
+            return collate.debug
+
+        if subcommand_name in ["destroy"]:
+            return Destroyer(prj)(subcommand_args)
 
-def main() -> None:
-    parser = pydantic_argparse.ArgumentParser(
+        use_pipestat = (
+            prj.pipestat_configured_project
+            if getattr(subcommand_args, "project", None)
+            else prj.pipestat_configured
+        )
+
+        if subcommand_name in ["table"]:
+            if use_pipestat:
+                return Tabulator(prj)(subcommand_args)
+            else:
+                raise PipestatConfigurationException("table")
+
+        if subcommand_name in ["report"]:
+            if use_pipestat:
+                return Reporter(prj)(subcommand_args)
+            else:
+                raise PipestatConfigurationException("report")
+
+        if subcommand_name in ["link"]:
+            if use_pipestat:
+                Linker(prj)(subcommand_args)
+            else:
+                raise PipestatConfigurationException("link")
+
+        if subcommand_name in ["check"]:
+            if use_pipestat:
+                return Checker(prj)(subcommand_args)
+            else:
+                raise PipestatConfigurationException("check")
+
+        if subcommand_name in ["clean"]:
+            return Cleaner(prj)(subcommand_args)
+
+        if subcommand_name in ["inspect"]:
+            from warnings import warn
+
+            warn(
```

Comment:
this is incorrect. looper inspect should still work.

Reply:
Ok, I added back the command `inspect_project` from Eido with a TODO to implement functionality that inspects the looper config. I also added a placeholder (but skipping) test for now.